reader/processor/processor.go

```diff
@@ -120,6 +133,39 @@ func ProcessEntryWebPage(entry *model.Entry) error {
 	return nil
 }
 
+func fetchYouTubeWatchTime(url string) int {
+	clt := client.NewClientWithConfig(url, config.Opts)
+	response, browserErr := browser.Exec(clt)
+	if browserErr != nil {
+		return 0
```

Comment:
Should the function returns 0 or an error if the scraping fail?

Having something in the logs that indicate the request failed would be nice. I would return an error here and `ProcessFeedEntries()` could display the error in the logs.